iac-common/src/test/java/org/sonar/iac/common/extension/IacSensorTest.java

```diff
@@ -65,7 +65,14 @@
 class IacSensorTest extends AbstractSensorTest {
 
   TreeParser<Tree> testParser = (source, inputFileContext) -> {
-    throw new RuntimeException();
+    throw new RuntimeException("RuntimeException message");
+  };
```

Comment:
I would rename this variable to `testParserThrowsRuntimeException`, so that we directly/clearly understand on usage that this is not gonna parse anything but will just throw an error.